README.md

```diff
@@ -1,30 +1,30 @@
-# Micronaut XXX
+# Micronaut Pulsar
 
-[![Maven Central](https://img.shields.io/maven-central/v/io.micronaut.xxx/micronaut-xxx.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22io.micronaut.xxx%22%20AND%20a:%22micronaut-xxx%22)
-[![Build Status](https://github.com/micronaut-projects/micronaut-xxx/workflows/Java%20CI/badge.svg)](https://github.com/micronaut-projects/micronaut-xxx/actions)
-
-Micronaut XXX is...
+[![Maven Central](https://img.shields.io/maven-central/v/io.micronaut.xxx/micronaut-pulsar.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22io.micronaut.xxx%22%20AND%20a:%22micronaut-pulsar%22)
+[![Build Status](https://github.com/micronaut-projects/micronaut-pulsar/workflows/Java%20CI/badge.svg)](https://github.com/micronaut-projects/micronaut-pulsar/actions)
 
+Micronaut Pulsar is integration between Apache Pulsar messaging system and Micronaut framework.
+## WORK IN PROGRESS
 ## Documentation
 
-See the [Documentation](https://micronaut-projects.github.io/micronaut-xxx/latest/guide/) for more information. 
+See the [Documentation](https://micronaut-projects.github.io/micronaut-pulsar/latest/guide/) for more information. 
 
-See the [Snapshot Documentation](https://micronaut-projects.github.io/micronaut-xxx/snapshot/guide/) for the current development docs.
+See the [Snapshot Documentation](https://micronaut-projects.github.io/micronaut-pulsar/snapshot/guide/) for the current development docs.
 
 ## Examples
 
-Examples can be found in the [examples](https://github.com/micronaut-projects/micronaut-xxx/tree/master/examples) directory.
+Examples can be found in the [examples](https://github.com/micronaut-projects/micronaut-pulsar/tree/master/examples) directory.
```

Comment:
Examples are made in the docs directly. I didn't have much time at that point to explore and understand how to use external examples and inject them. Didn't know this was that important for first(beta/priview) phase as docs do contain some examples but only in Java.

Reply:
It would be super helpful to get this in SNAPSHOT or something. That way I could make easy examples plus I bet it would be more exposed so other developers could give advises or bug reports. I'm stuck on this pull request for about 20 days now.

Reply:
@graemerocher WDYT about creating some RC version of this project, so we can move forward?

Reply:
@pgressa Thanks and sorry for bothering but I cannot put example and dependency on SNAPSHOT as I cannot push to OSS snapshots nor will this be pushed (before merge to master I guess). So if I put examples with dependency on `project(:..)` it will not work if someone downloads it or so and I cant have it with `io.micronat.pulsar` as it's not in snapshot nor release yet.